README.md

```diff
@@ -0,0 +1,72 @@
+# Visualization API
+
+This repository contains app that fetch (serverless) **BIRAFFE2** dataset and then visualizate it.
+
+The **"BIRAFFE2: The 2nd Study in Bio-Reactions and Faces for Emotion-based Personalization for AI
+Systems"** dataset is part of the work carried out by the AfCAI Research Group in Kraków, Poland.
+
+**BIRAFFE2** source: https://zenodo.org/record/3865860#.XvjpwecwhPY
+
+## Table of Contents
+
+- [Table of Contents](#table-of-contents)
+- [Getting started](#getting-started)
+- [Scripts](#scripts)
+
+## Getting started
+
+### Clone the repository
+
+```bash
+git clone https://github.com/kuzmialke-and-friends/visualization-api
+```
+
+### Go inside the directory
+
+```bash
+cd visualization-api
+```
```

Comment:
Let's merge some of them.
```suggestion
### Clone the repository anad go inside it

```bash
git clone https://github.com/kuzmialke-and-friends/visualization-api
cd visualization-api
```